src/foam/nanos/theme/Theme.js

```diff
@@ -519,7 +519,18 @@ foam.CLASS({
       name: 'spid',
       class: 'Reference',
       of: 'foam.nanos.auth.ServiceProvider',
-      value: foam.nanos.auth.ServiceProviderAware.GLOBAL_SPID
+      documentation: `
+        Only set spid on the main theme so that it can be looked up via spid
+        when could not find theme via domain.
+
+        By default, theme.spid is empty to denote an auxiliary theme which is
+        used for augmenting the theme for specific users and groups.
```

Comment:
What do you mean by "augmenting"?

Reply:
Themes.findTheme(x) tries to find theme by domain, if not found then find by user.SPID, if cannot get one then make a new theme. User and group can be configured with a theme to augment the found/created theme previously.

Reply:
It checks for user and group theme then does copyFrom instead of replacing. 

https://github.com/foam-framework/foam2/blob/927d3190f9c2b256b180a247ba1deb2ca3955b6f/src/foam/nanos/theme/Themes.js#L186-L203

Reply:
Fixed https://github.com/foam-framework/foam2/pull/4125/commits/9a9d9c5c4f163a1545165f77e926b5ff08030465 - use "*" as default spid for theme.